src/Repository/Schedule/InterviewQuestion/InterviewQuestionRepository.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository\Schedule\InterviewQuestion;
+
+use App\Entity\InterviewQuestion;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+final class InterviewQuestionRepository extends ServiceEntityRepository implements InterviewQuestionRepositoryInterface
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, InterviewQuestion::class);
+    }
+
+    public function createInterviewQuestion(string $position): InterviewQuestion
```

Comment:
What "position" stand for? Could it be a mistake and you wanted to write "question"?